File: validation/validation-annotation-processor/src/test/java/io/koraframework/validation/annotation/processor/testdata/ValidTaz.java
package io.koraframework.validation.annotation.processor.testdata;

import io.koraframework.validation.common.annotation.Pattern;
import io.koraframework.validation.common.annotation.Valid;

@Valid
public record ValidTaz(@Pattern("\\d+") String number) {

    public static final String IGNORED = "ops";
}

File: validation/validation-symbol-processor/src/test/kotlin/io/koraframework/validation/symbol/processor/testdata/ValidTaz.kt
package io.koraframework.validation.symbol.processor.testdata

import io.koraframework.validation.common.annotation.Pattern
import io.koraframework.validation.common.annotation.Valid

@Valid
data class ValidTaz(@Pattern("\\d+") val number: String) {

    companion object {
        const val ignored: String = "ops"
    }
}
